CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- `is_imbalanced` flag added to `__init__`
```

Comment:
Please follow previous pattern, something like this :
- `is_imbalanced` parameter added to ConfusionMatrix `__init__` method (`### Changed` section)